mypy/messages.py

```diff
@@ -855,6 +855,11 @@ def unsupported_type_type(self, item: Type, context: Context) -> None:
     def redundant_cast(self, typ: Type, context: Context) -> None:
         self.note('Redundant cast to {}'.format(self.format(typ)), context)
 
+    def unimported_type_becomes_any(self, prefix: str, typ: Type, ctx: Context) -> None:
+        self.fail("{} becomes {} due to an unfollowed import (such imports occur either "
```

Comment:
I agree -- this error message is too long.  I also agree that it'll probably be somewhat confusing.  I'd say: remove the parenthetical explanation for now, and we can consider some other mechanism of explaining to the user if we think it's a big enough problem.  (But we can/should do that in another PR.)  We could consider outputting one `Note:` at the end if there are any of these errors explaining what an unfollowed import is.